### DISCUSSION TITLE
3rd party SDKs

### DISCUSSION THREAD

**Discussion by mirkotoday** [skiptools/skip#210] [Q&A] [answered]
Hello skip-Team,

How does Skip support the implementation of 3rd-party-SDKs and Libraries such as Google Analytics, NewRelic, Dynatrace, Adobe Target, OneTrust, Tealium etc.?

Thanks a lot,
Mirko

**Comment by marcprux** [comment, marked as answer]
Skip supports integrating with any 3rd-party SDK through adding dependencies separately on the Swift and Kotlin sides of the app. See https://skip.tools/docs/dependencies/#implementation

We have a some community integration modules that do this for you out of the box, such as the [Firebase framework](https://skip.tools/docs/modules/skip-firebase/). For other libraries that don't yet have any community integration frameworks available, you can write your own (and hopefully contribute it to the community!)
